src/screens/consul/LoginRegister/ConsulLoginScreen.js

```diff
@@ -34,42 +35,51 @@ const showLoginFailAlert = () =>
   Alert.alert(texts.consul.loginFailedTitle, texts.consul.loginFailedBody);
 
 export const ConsulLoginScreen = ({ navigation }) => {
-  const [secureTextEntry, setSecureTextEntry] = useState(true);
-  const [registrationLoading, setRegistrationLoading] = useState(false);
+  const [isSecureTextEntry, setIsSecureTextEntry] = useState(true);
+  const [isLoginLoading, setIsLoginLoading] = useState(false);
 
   const {
     control,
     formState: { errors },
     handleSubmit
-  } = useForm();
+  } = useForm({
+    defaultValues: {
+      email: null,
+      password: null
```

Comment:
does react hook form like `null` more than `''`?

Reply:
The documentation states that you can use null or empty strings.

`defaultValues` Note: It is encouraged that you set a defaultValue for all inputs to non-undefined such as the empty string or null.

[React-Hook-Form v6](https://react-hook-form.com/v6/api/)

Reply:
ah ok. good.
what do you think about using empty string for string fields and null for others?

Reply:
All code needs to stay in the same structure. I will update my code with empty string.

Reply:
do you mean updating empty strings just for string values or for everything? i think we should not have default values with empty strings, if the values at the end are numbers for examples.